docs/source/en/guides/upload.md

```diff
@@ -435,62 +435,8 @@ For more detailed information, take a look at the [`HfApi`] reference.
-      For example, json files can be merged into a single jsonl file, or large datasets can be exported as Parquet files.
-    - The maximum number of files per folder cannot exceed 10k files per folder. A simple solution is to
-      create a repository structure that uses subdirectories. For example, a repo with 1k folders from `000/` to `999/`, each containing at most 1000 files, is already enough.
-- **File size**: In the case of uploading large files (e.g. model weights), we strongly recommend splitting them **into chunks of around 5GB each**.
-There are a few reasons for this:
-    - Uploading and downloading smaller files is much easier both for you and the other users. Connection issues can always
-      happen when streaming data and smaller files avoid resuming from the beginning in case of errors.
-    - Files are served to the users using CloudFront. From our experience, huge files are not cached by this service
-      leading to a slower download speed.
-In all cases no single LFS file will be able to be >50GB. I.e. 50GB is the hard limit for single file size.
-- **Number of commits**: There is no hard limit for the total number of commits on your repo history. However, from
-our experience, the user experience on the Hub starts to degrade after a few thousand commits. We are constantly working to
-improve the service, but one must always remember that a git repository is not meant to work as a database with a lot of
-writes. If your repo's history gets very large, it is always possible to squash all the commits to get a
-fresh start using [`super_squash_history`]. This is a non-revertible operation.
-- **Number of operations per commit**: Once again, there is no hard limit here. When a commit is uploaded on the Hub, each
-git operation (addition or delete) is checked by the server. When a hundred LFS files are committed at once,
-each file is checked individually to ensure it's been correctly uploaded. When pushing data through HTTP with `huggingface_hub`,
-a timeout of 60s is set on the request, meaning that if the process takes more time, an error is raised
-client-side. However, it can happen (in rare cases) that even if the timeout is raised client-side, the process is still
-completed server-side. This can be checked manually by browsing the repo on the Hub. To prevent this timeout, we recommend
-adding around 50-100 files per commit.
-
-### Practical tips
-
-Now that we've seen the technical aspects you must consider when structuring your repository, let's see some practical
-tips to make your upload process as smooth as possible.
+
+Check out our [recommendations](https://huggingface.co/docs/hub/repositories-recommendations) to learn more about the best practices on how to structure your repositories on the Hub. Once you have in mind the technical aspects you must consider, let's move one with some practical tips to make your upload process as smooth as possible.
```

Comment:
I think specifying the name of the guide makes it clearer what to expect for users when they click on it

```suggestion
Check out our [Repository limitations and recommendations](https://huggingface.co/docs/hub/repositories-recommendations) guide for best practices on how to structure your repositories on the Hub. Next, let's move on with some practical tips to make your upload process as smooth as possible.
```

Reply:
Thanks for the suggestion, as always :heart: